package.json

```diff
@@ -4,22 +4,23 @@
   "description": "ETL to extract data from Syndication and store as JSON",
   "main": "app.js",
   "scripts": {
```

Comment:
Can the scripts section be alphabetised to aid finding scripts and identifying naming conventions.